fnet/cli/train_model.py

```diff
@@ -92,27 +92,27 @@ def main(args: Optional[argparse.Namespace] = None) -> None:
         parser = argparse.ArgumentParser()
         add_parser_arguments(parser)
         args = parser.parse_args()
-    if not os.path.exists(args.json):
+    if not args.json.exists():
```

Comment:
Do you want to add the same `if args.json and not args.json.exists()` here as you did in the `predict.py`?

```suggestion
    if args.json and not args.json.exists():
```